libcloud/dns/drivers/gandi.py

```diff
@@ -147,16 +147,20 @@ def delete_zone(self, zone):
         return res.object
 
     def _to_record(self, record, zone):
+        extras = {'ttl': record['ttl']}
+        value = record['value']
+        if record['type'] == 'MX':
+            extras['priority'] = record['value'].split(' ')[0]
+            value = record['value'].split(' ')[1]
```

Comment:
`record['value']` looks like `15 aspmx.l.google.com` - I didn't try it, but I expect leaving the value as that raw value would spit out `...15 15 aspmx...` when converted to BIND speak.

Given that I don't have an exhaustive knowledge of DNS record types off the top of my head (or how Gandi returns them all), I expect this is an incomplete patch that happened to work for the (or my) Gandi MX case.  I'm not sure how your team prefers to handle that, but I did want it to at least start on the way in so folks (or just me) don't have to monkey patch locally.